pobieranie_zdjęć_z_URL.ipynb

```diff
@@ -0,0 +1,235 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "authorship_tag": "ABX9TyNiuW/ybY0X6cfaoiHqqeWI",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/PiotrMaciejKowalski/BigData2022-actors/blob/pobieranie_zdj%C4%99%C4%87/pobieranie_zdj%C4%99%C4%87_z_URL.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Przed uruchomieniem trzeba utworzyć folder \"ImgData\" w MyDrive\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "PrZUQucODnQz"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install pyspark py4j\n",
+        "!pip install -q findspark\n",
+        "!apt-get install openjdk-8-jdk-headless -qq > /dev/null\n",
+        "!wget https://dlcdn.apache.org/spark/spark-3.3.1/spark-3.3.1-bin-hadoop2.tgz\n",
+        "!tar xf spark-3.3.1-bin-hadoop2.tgz\n",
+        "import pyspark\n",
+        "import findspark\n",
+        "from pyspark.sql import SparkSession\n",
+        "import os\n",
+        "import pyspark.sql.types as T\n",
+        "from pyspark.sql.functions import split, col, monotonically_increasing_id, when\n",
+        "from pyspark.sql.types import StructType, StringType, IntegerType, BooleanType, FloatType, TimestampType, DateType, ArrayType, MapType, StructField\n",
+        "os.environ[\"JAVA_HOME\"] = \"/usr/lib/jvm/java-8-openjdk-amd64\"\n",
+        "os.environ[\"SPARK_HOME\"] = \"/content/spark-3.3.1-bin-hadoop2\"\n",
+        "spark=SparkSession.builder.appName('Colab').getOrCreate()"
```

Comment:
użyj proszę funkcji, która ładuje sparka. Została ona utworzona w pliku lib/pyspark_init.py - funkcja create_spark_context. Możesz podejrzeć np. w notatniku Wczytanie_danych jak to zrobić

Reply:
zrobione